harvester_e2e_tests/fixtures/api_client.py

```diff
@@ -70,6 +70,12 @@ def rancher_wait_timeout(request):
     return request.config.getoption("--rancher-cluster-wait-timeout", 1800)
 
 
+@pytest.fixture(scope="session")
+def ubuntu_checksum(request):
```

Comment:
we already put the checksum in class `ImageInfo`, it looks we would not need this fixture.

Reply:
It allows you to set the checksum on the tests that don't use a image_info object, like `create_image_url`. It's probably worth refactoring that at some point to use the image_info object, but it's out of scope for this PR.